docs/styled.md

```diff
@@ -197,3 +197,27 @@ render(
   </Container>
 )
 ```
+
+### `as` prop
+
+To use styles from a styled component but change the element that's rendered, you can use the `as` prop.
```

Comment:
It would be great to explain under this section how prop filtering works and how you might change it with custom `shouldForwardProp`